frontend/model/notifications/mainNotificationsMixin.js

```diff
@@ -125,38 +125,33 @@ const periodicNotificationEntries = [
     notificationData: {
       stateKey: 'expiringProposals',
       emitCondition ({ rootGetters }) {
-        this.expiringProposalsByGroup = rootGetters.groupsByName.map(group => {
-          const { contractID } = group
-          const expiringProposals = []
-          const groupProposals = rootGetters.groupProposals(contractID) || {}
-          for (const proposalId in groupProposals) {
-            const proposal = groupProposals[proposalId]
-            if (proposal.status === STATUS_OPEN &&
-              !proposal.notifiedBeforeExpire &&
-              proposal.data.expires_date_ms < (Date.now() + DAYS_MILLIS)) {
-              expiringProposals.push({
-                proposalId,
-                proposalType: proposal.data.proposalType,
-                proposalData: proposal.data.proposalData,
-                expires_date_ms: proposal.data.expires_date_ms,
-                createdDate: proposal.meta.createdDate,
-                creator: proposal.meta.username
-              })
-            }
+        const groupProposals = rootGetters.currentGroupState.proposals || {}
```

Comment:
Hmm.. why was this modified? Doesn't it make sense for us to emit notifications for other groups we're signed into at the moment that have expiring proposals?

Reply:
@taoeffect It's due to the way that `gi.notifications/emit` works.
it sends the notification for the group that the user's has logged in on at the time but has no capability to send it to the other groups that are not active. (reference [here](https://github.com/okTurtles/group-income/blob/4675aab76b63524c05483e0ab1d890aa5105f24a/frontend/model/notifications/selectors.js#L15))

Reply:
Why do you say it "has no capability to send it to the other groups that are not active"?

I looked at your link, it doesn't suggest that it can't support notifications for other groups. That's why the `NotificationData` object takes a `groupID` parameter

Reply:
You are right. made additional changes accordingly.